justfile

```diff
@@ -30,4 +33,16 @@ ruff *args:
   docker compose exec app ruff format src
 
 lint:
-  just ruff --fix
\ No newline at end of file
+  just ruff --fix
+
+backup:
+  docker compose exec app_db scripts/backup
+
+# examples:
+# "just get-backup dump_name_2021-01-01..backup.gz" to copy particular backup
+# "just get-backup" to copy directory (backups) with all dumps
+get-backup *args:
```

Comment:
get-beckups sounds not much explicit. get from where?
let's add a more explicit naming like `extract-docker-backup`, `mount-docker-backup`, etc.